Makefile

```diff
@@ -165,11 +165,11 @@ $(CONTROLLER_GEN): $(VENDOR_MODULES)
 	mkdir -p $(@D)
 	$(GO) build -o $@ sigs.k8s.io/controller-tools/cmd/controller-gen
 
-deploy/crds/submariner.io_servicediscoveries.yaml: ./api/submariner/v1alpha1/servicediscovery_types.go $(VENDOR_MODULES) | $(CONTROLLER_GEN)
+deploy/crds/submariner.io_servicediscoveries.yaml: api/v1alpha1 $(VENDOR_MODULES) | $(CONTROLLER_GEN)
 	$(CONTROLLER_GEN) $(CRD_OPTIONS) paths="./..." output:crd:artifacts:config=deploy/crds
 	test -f $@
 
-deploy/crds/submariner.io_brokers.yaml deploy/crds/submariner.io_submariners.yaml: ./api/submariner/v1alpha1/submariner_types.go $(VENDOR_MODULES) | $(CONTROLLER_GEN)
+deploy/crds/submariner.io_brokers.yaml deploy/crds/submariner.io_submariners.yaml: api/v1alpha1 $(VENDOR_MODULES) | $(CONTROLLER_GEN)
```

Comment:
This needs to list specific files, not `api/v1alpha1` as a whole.

Reply:
Ya. This was IDE being extra helpful :)